org.cvfosammmm.Setzer.json

```diff
@@ -0,0 +1,98 @@
+{
+    "app-id": "org.cvfosammmm.Setzer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
```

Comment:
Have you tried bumping up this to 3.34?

Reply:
Unfortunately that didn't work. The TeXLive extension would not be detected when I tried.

Reply:
Could it be because TeXLive is labeled "18.08", and gnome 3.34 might assume extension are "19.08" (current version of freedesktop Sdk)?

Reply:
3.32 should be also based on 18.08.

Reply:
I changed it to 3.32.